### ISSUE TITLE
api_definition.definition.enabled is defined in the Classic API versioning document even if it should not be used

### ISSUE THREAD

**Issue by kaye-at-tyk** [TykTechnologies/tyk-docs-hugo#5288]
## Branch/Environment/Version
- Tyk Documentation: v5.5 (Latest)
- Tyk Gateway & Dashboard: v5.3.1 & v5.5.0
- Note: `api_definition.definition.enabled` is populated by default (set to `false`) when creating Classic APIs using versions above

## Relevant Document
https://site-dev.tykbeta.com/docs/tyk-gateway-api/api-definition-objects/

## Describe the request
- The document states that Classic APIs have an API-level configuration (`api_definition.definition.enabled`) to determine if it's using versioning or not:

> There is also some API-level configuration for versioning, which is located in the definition section of the Tyk Classic API definition:
> The definition section has the following fields:
> ...
> enabled: toggle to enable or disable versioning

- According to the answer in this [Tyk StackOverflow question](https://stackoverflowteams.com/c/tyk-technologies/questions/3225), `api_definition.definition.enabled` does not apply for Classic APIs
- This has caused some confusion for a customer who tried to set `api_definition.definition.enabled` to `true` in their Classic API (reference: [Zendesk ticket 19811](https://tyksupport.zendesk.com/agent/tickets/19811))
- As `api_definition.definition.enabled` should not be used for Classic APIs, this should either be omitted from the document, or should be marked as "not applicable"

**Comment by andyo-tyk** [comment]
This will be addressed by https://github.com/TykTechnologies/tyk-docs/pull/5293

**Comment by sharadregoti** [comment]
We have addressed this (by removing the statement) in the latest reference document of [Tyk Classic APIs](https://tyk.io/docs/api-management/gateway-config-tyk-classic/#tyk-classic-api-versioning).

@andyo-tyk, can you confirm this as well?

<img width="723" alt="Image" src="https://github.com/user-attachments/assets/f6e67148-07d2-4091-95e9-9a8f6aa3f6e7" />

**Comment by andyo-tyk** [comment]
Yes, when we reworked the Tyk API definition section for 5.8 (to prioritise Tyk OAS), the changes in 5293 were moved to https://tyk.io/docs/api-management/gateway-config-tyk-classic/#tyk-classic-api-versioning

The current docs are correct.
